docs/pages/dsFPD/ds-front-panel-display_L3_TestProcedure.md

```diff
@@ -0,0 +1,470 @@
+
+Once the environment is set up, you can execute the test cases with the following command
+
+```bash
+python <TestCaseName.py> --config </PATH>/ut/host/tests/configs/example_rack_config.yml --deviceConfig </PATH>/ut/host/tests/configs/deviceConfig.yml
+```
+## Test Cases
+
+### dsFPD_test01_EnableDisableAndVerifyLEDIndicators.py
+
+#### Platform Support - test01
+
+- Source
+- Sink
+
+#### User Input Required - test01
+
+**Yes**: User interaction is necessary to confirm front panel LED status (This will be automated later).
+
+#### Acceptance Criteria - test01
+
+Set supported Front panel LED State to ON and OFF and verify.
+
+#### Expected Results - test01
+
+The test enables the supported front panel LED, and subsequently disabled the LED
+
+Success Criteria
+
+- User should see the supported LED turned to ON State
+- User should see the supported LED turned to OFF state.
+
+#### Test Steps - test01
+
+- Initiate the Test:
+
+  - Select and execute the Python file: **`dsFPD_test01_EnableDisableAndVerifyLEDIndicators.py`**
+  - The test will automatically download all required artifacts, copying them to the designated target directory before commencing execution.
+
+- LED state Verification:
+
+    The test will enable LED and prompt the user with the following:
+
+  - Question: "Is Indicator state dsFPD_STATE_ON ? (Y/N):? (Y/N)"
+  - Press **Y** if specified LED indicator is ON  (this will mark the step as PASS).
+  - Press **N** if specified LED indicator is OFF (this will mark the step as FAIL).
+
+- LED status confirmation (LED indicator Disabled):
+
+  After confirming LED state, the test will disable the LED indicator and prompt the user again:
+
+  - Question: "Is Indicator state dsFPD_STATE_OFF ? (Y/N)"
+  - Press **Y** if specified LED indicator is OFF (this will mark the step as PASS).
+  - Press **N** if specified LED indicator is ON  (this will mark the step as FAIL).
+
+- Repeat for All supported LED indicators:
+
+  The test will iterate through all supporeted Indicators, enabling/disabling each one and collecting user feedback accordingly.
+
+- Test Conclusion:
+
+  Upon receiving user responses for all indicators, the test will conclude and present a final result: PASS or FAIL based on the user inputs throughout the test execution.
+
+### dsFPD_test02_SetVerifyLEDIndicatorsBrightness.py
+
+#### Platform Support - test02
+
+- Source
+- Sink
+
+#### User Input Required - test02
+
+**Yes**: User interaction is necessary to confirm front panel LED status and Brightness (This will be automated later).
+
+#### Acceptance Criteria - test02
+
+Set supported Front panel LED can be set with different brightness, 0%, 50% and 100%
+
+#### Expected Results - test02
+
+The test enables the supported front panel LED, sets brightness value to 0, 50 and 100, subsiquently disables the LED indicator.
+
+Success Criteria
+
+- User should see the supported LED turned to ON State
+- User shoudl see the supported LED glowing in 0% brightness
```

Comment:
spell mistakes.. should not "shoudl"  all through next 3 lines.

Reply:
updated